src/commands/functions/create.js

```diff
@@ -404,20 +434,8 @@ const installAddons = async function (context, functionAddons, fnPath) {
         log,
         error,
       })
-      if (addonCreated && addonDidInstall) {
-        await addEnvVariables(api, site)
-        const { confirmPostInstall } = await inquirer.prompt([
-          {
-            type: 'confirm',
-            name: 'confirmPostInstall',
-            message: `This template has an optional setup script that runs after addon install. This can be helpful for first time users to try out templates. Run the script?`,
-            default: false,
-          },
-        ])
-        if (confirmPostInstall) {
-          addonDidInstall(fnPath)
-        }
-      }
+
+      await handleAddonDidInstall({ addonCreated, addonDidInstall, context, fnPath })
```

Comment:
There is some optimisation to be done here as `handleAddonDidInstall` will read `siteData` and `siteAddons` again internally, but that would require more refactoring.